src/routes/index.js

```diff
@@ -1,14 +1,16 @@
-const React = require('react');
+const { lazy: Lazy } = require('react');
 const Auth = require('./auth');
-const Layout = require('../containers/Layout');
+// const Layout = require('../containers/Layout');
+const Layout = require('../components/Layout');
 const NotFoundPage = require('../components/NotFoundPage');
 const NotFoundHelpers = require('./helpers/not-found');
 const HomePage = require('./home/components/HomePage');
 const SignupPage = require('./join/containers/SignupPage');
-const LoginPage = require('./login/containers/LoginPage');
+// const LoginPage = require('./login/containers/LoginPage');
+const LoginPage = require('./login/components/LoginPage');
```

Comment:
In case this was leftover from testing things out:
```suggestion
const LoginPage = require('./login/containers/LoginPage');
```